code/script.js

```diff
@@ -1,53 +1,231 @@
-// DOM selectors (variables that point to selected DOM elements) goes here 👇
+// All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const sendButton = document.getElementById('send-btn')
+const nameInput = document.getElementById('name-input')
+const form = document.getElementById('name-form')
 
-// Functions goes here 👇
 
-// A function that will add a chat bubble in the correct place based on who the sender is
+// Global variables, if you need any, declared here
+
+let currentQuestionNumber = 1
+let saveEvent = ""  // Saves a value outside the function
+
+// Starts here
+const greeting = () => {
+  showMessage(`Hello there, My name is JetSetBot.  What's your name?`, 'bot')
+}
+setTimeout(greeting, 500)
+
+
+// Set up your eventlisteners here
+sendButton.addEventListener('click', (event) => {
+  event.preventDefault();
+  currentQuestion();
+});
+
+// Functions declared here
+// DECIDES WHICH QUESTION WE ARE HANDLING
+const currentQuestion = () => {
+  if (currentQuestionNumber === 1) {
+    handleNameInput();
+  }
+  else if (currentQuestionNumber === 2) {
+    classQuestion();
+  }
+  else if (currentQuestionNumber === 3) {
+    timeQuestion();
+  }
+  else if (currentQuestionNumber === 4) {
+    confirmQuestion();
+  }
+  else if (currentQuestionNumber === 5) {
+    askForEmail();
```

Comment:
be careful here, if the user declines do you still wanna ask for the e-mail address? Right not that't the case